DDG4/python/DDG4.py

```diff
@@ -131,14 +131,29 @@ def _registerGlobalFilter(self, filter):
   self.get().registerGlobalFilter(Interface.toAction(filter))
 
 
+def _evalProperty(data):
+  """
+    Function necessary to extract real strings from the property value.
+    Strings may be emraced by quotes: '<value>'
+  """
+  try:
+    if isinstance(data, str):
+      return eval(data)
```

Comment:
```suggestion
      import ast
      return ast.literal_eval(data)
   except ValueError:
```
https://docs.python.org/3.7/library/ast.html#ast.literal_eval
Safely(!) eval 
```
>>> import ast
>>> ast.literal_eval('foo')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/ast.py", line 80, in literal_eval
    return _convert(node_or_string)
  File "/usr/lib64/python2.7/ast.py", line 79, in _convert
    raise ValueError('malformed string')
ValueError: malformed string
>>> ast.literal_eval('"foo"')
'foo'
```